api/rosetta/options_integration_test.go

```diff
@@ -70,124 +70,118 @@ func TestAPI_Options(t *testing.T) {
 
 	assert.True(t, options.Allow.HistoricalBalanceLookup)
 
-	if assert.Len(t, options.Allow.OperationStatuses, 1) {
+	require.Len(t, options.Allow.OperationStatuses, 1)
```

Comment:
I'll stop repeating myself now 😄 But here I especially don't like it.
One would need to go through the rabbit hole and fix one issue after another, since the `require` would halt the execution, vs having the full list of errors at once and go through them one by one.

Reply:
I feel like it's exactly the same as with `require.NoError(t, err)` right before other assertions. It cuts the test directly even though the test might have been able to proceed, but it's fine. What's the difference, after all? :p